receiver/postgresqlreceiver/metadata.yaml

```diff
@@ -19,49 +19,49 @@ attributes:
 metrics:
   postgresql.blocks_read:
     description: The number of blocks read.
-    units: 1
+    unit: 1
```

Comment:
The unit `1` is confusing to me, is this specified anywhere?

Reply:
Yes, it's defined here and there, e.g. https://github.com/open-telemetry/opentelemetry-specification/tree/main/specification/metrics/semantic_conventions#instrument-units

We here an issue to define it uniformly https://github.com/open-telemetry/opentelemetry-specification/issues/705